solidity/contracts/token/bridge/EverclearTokenBridge.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.22;
+
+import {ITokenBridge, Quote} from "../../interfaces/ITokenBridge.sol";
+import {HypERC20Collateral} from "../HypERC20Collateral.sol";
+import {IEverclearAdapter} from "../../interfaces/IEverclearAdapter.sol";
+
+/**
+ * @title EverclearTokenBridge
+ * @author Hyperlane Team
+ * @notice A token bridge that integrates with Everclear's intent-based architecture
+ * @dev Extends HypERC20Collateral to provide cross-chain token transfers via Everclear's intent system
+ */
+contract EverclearTokenBridge is HypERC20Collateral {
```

Comment:
we should not inherit the `handle` side of this contract nor the movable collateral logic

Reply:
Removed